vignettes/site_only/deploy_model.Rmd

```diff
@@ -0,0 +1,189 @@
+                                  database = "SAM")
+
+con <- dbConnect(odbc::odbc(), .connection_string = my_con)
+
+train_query <- "SELECT 
+                 [FacilityAccountID]
+                ,[AdmitAgeNBR]
+                ,[GenderCD]
+                ,[EthnicGroupDSC]
+                ,[MaritalStatusDSC]
+                ,[Readmit30FLG]
+  FROM [SAM].[Schema].[SummaryVisitBASE]
+  WHERE InpatientFLG = 'F'"
+
+d <- db_read(con, train_query)
+
+models <- machine_learn(d = d, 
+                        FacilityAccountID,
+                        models = "rf",
+                        outcome = Readmit30FLG)
+
+# Save my model
+save(models, file = "readmit_models.RDA")
+```
+
+# Pushing predictions to a database
+
+While the frequency will vary depending on the use case, for most healthcare ML scenarios you'll need predictions on a reoccurring basis. Daily batch predictions are common. To create predictions, new rows (representing patients or patient encounters) are run against the model that lives in the RDA file. 
+
+Instead of slimply creating predictions in R, however, in a production scenario you'll be pushing these predictions to a database. Typically we append to the same table each night, such that a person will get an updated risk score for each day they're in the hospital (for example).
+
+## Creating predictions
+
+Now we focus on the code that will run each night to create predictions. Note a couple of things:
+
+- The query is different from the development step--here you're only querying patients that need a prediction
+- We read the model from disk; if needed, a web service could be involved
+
+
+```{r}
+prod_query <- "SELECT 
+                 [FacilityAccountID]
+                ,[AdmitAgeNBR]
+                ,[GenderCD]
+                ,[EthnicGroupDSC]
+                ,[MaritalStatusDSC]
+                ,[Readmit30FLG]
+  FROM [SAM].[Schema].[SummaryVisitBASE]
+  WHERE InpatientFLG = 'T'"
+
+d_out <- db_read(con, prod_query)
+
+load("readmit_models.RDA")
+
+predictions <- predict(models, d_out)
+```
+
+## Create the output table
+
+Before we create the output table, look at the format of your output by typing `glimpse(predictions)`
+
+You might want push fewer columns to a table each night. Here's how you slim this down:
+
+```{r}
+predictions <- predictions %>% 
+  select(FacilityAccountID, predicted_Readmit30FLG) %>% # Grab just two columns
+  rename(PredictedProbability = predicted_Readmit30FLG) # Change col name
+```
+
+Now that we have only the columns necessary, let's and populate the output table
+
+### Outside of a Health Catalyst environment
+
+Use a tool like SQL Server Management Studio and a `CREATE TABLE` statement
+
+### Inside a Health Catalyst environment
+
+1. Append a few utility columns to your output
+
+```{r}
+predictions <- predictions %>%
+  add_SAM_utility_cols
+```
+
+2. Use Source Area Mart Designer (SAMD) to create the table (since this creates metadata)
```

Comment:
Maybe we could provide an example binding/entity name consistent with your code below so they get it right on the first try. Maybe something like: "If you're using SAMD to define your table, name the binding like MLOutputTable and ensure your entity name gets generated as expected (no need to add the suffix BASE in SAMD as that will happen automatically).